docs/guides.md

```diff
@@ -523,3 +523,89 @@ Now we enable autostart at the virtual machine level.
 5. Checking the output 
 `# xe vm-param-list uuid=<VM_UUID> | grep other-config`
 
+## Guest UEFI Secure Boot
```

Comment:
I don't think the whole secure boot section should be in the Guides page. `guest.md`, which is dedicated to VMs in general, appears more appropriate to me.

Reply:
Hmm... perhaps some restructuring is necessary.  `guest.md` appears to be exclusively for guest tools, which is a specific bundle of utilities, no?

Reply:
This is true. It was an unspoken idea of mine that the `guests.md` would later address more than just the tools and be about VMs in general, as the documentation software appears to force us to limit the number of different pages we create if we want the left side panel to remain readable. It's not set in stone though and would indeed require some restructuring of the page.

So maybe the Guides page is ok as a temporary place, until a possible future rework.